CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/potions/MCPotionRecipe.java

```diff
@@ -0,0 +1,19 @@
+package crafttweaker.mc1120.potions;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.item.IIngredient;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import crafttweaker.api.potions.IPotionType;
+import net.minecraft.potion.PotionHelper;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.recipes.IPotionRecipe")
+@ZenRegister
+public class MCPotionRecipe {
+
+    @ZenMethod
+    public static void add(IPotionType input, IIngredient ingredient, IPotionType output) {
```

Comment:
Why is that in a separate class and not in the brewing manager?

<https://github.com/CraftTweaker/CraftTweaker/blob/1.12/CraftTweaker2-API/src/main/java/crafttweaker/api/recipes/IBrewingManager.java>